x-pack/platform/plugins/shared/fleet/server/services/package_policies/experimental_datastream_features.ts

```diff
@@ -156,8 +156,8 @@ export async function handleExperimentalDatastreamFeatureOptIn({
 
     if (isSyntheticSourceOptInChanged) {
       sourceModeSettings = {
-        _source: {
-          ...(featureMapEntry.features.synthetic_source ? { mode: 'synthetic' } : {}),
+        source: {
```

Comment:
shouldn't we keep the `featureMapEntry.features.synthetic_source` condition?

Reply:
Yes you are right